src/Rules/Generate.hs

```diff
@@ -80,7 +81,7 @@ compilerDependencies stage =
 
 generatedDependencies :: Stage -> Package -> [FilePath]
 generatedDependencies stage pkg
-    | pkg   == compiler = compilerDependencies stage
+    | pkg   == compiler = compilerDependencies stage ++ ["inplace/lib/bin/ghc-split"]
```

Comment:
`["inplace/lib/bin/ghc-split"]` should better be moved to `compilerDependencies`, so we have everything in one place.